src/Listener/RestRequestListener.php

```diff
@@ -68,19 +87,21 @@ public function __construct(
     public function onKernelController($event)
     {
         $request = $event->getRequest();
+        // We need make sure firewall allows the client to make the request,
```

Comment:
I don't really get the comment here:
- if firewall does not allow to make the request, the code will not reach this place
- firewall is basically just one use-case, I'd communicate (if needed) that we also need to support custom controller options here